Core/src/org/sleuthkit/autopsy/machinesettings/UserMachinePreferences.java

```diff
@@ -175,7 +141,7 @@ public static String getTempDirectory() {
     /**
      * @return The user-specified custom temp directory path or empty string.
      */
-    public static String getCustomTempDirectory() {
+    public static String getCustomTempDirectoryProperty() {
```

Comment:
I know that the use of "Property" has been proposed as a way to make the intent of these methods more clear, but that is not consistent with our usage to this point. I think that we can achieve the same clarity without changing our style with Path getCustomTempDirectoryPath(). Note the return type.